libcoap-sys/src/wrapper.h

```diff
@@ -8,3 +8,4 @@
  */
 
 #include <coap3/coap.h>
+#include <coap_config.h>
```

Comment:
Trailing newline should be added here.

```suggestion
#include <coap_config.h>

```